entrypoint.sh

```diff
@@ -31,6 +31,11 @@ if [[ "${INPUT_PRESET}" ]]; then
   pint_command+=" --preset ${INPUT_PRESET}"
 fi
 
+if [[ "${INPUT_ONLYDIFF}" ]]; then
+  pint_command+=" --diff=${INPUT_ONLYDIFF}"
+  INPUT_ONLYDIRTY=false
+fi
```

Comment:
About my comment https://github.com/aglipanci/laravel-pint-action/issues/21#issuecomment-2613791867
```suggestion
if [[ "${INPUT_ONLYDIFF}" == true ]]; then
  INPUT_ONLYDIFF=**assign the default branch**
fi

if [[ "${INPUT_ONLYDIFF}" ]]; then
  pint_command+=" --diff=${INPUT_ONLYDIFF}"
  INPUT_ONLYDIRTY=false
fi
```

Reply:
Hi, I also thought about that idea, but I discarded it.

>`--diff=[branch]`
https://github.com/laravel/docs/blob/ecc940f39ff2e052a1eafac1edb70777754f747c/pint.md?plain=1#L58-L62

Apparently the branch must always be declared, and I can't leave a default one because it's not a standard, 
the default branch depending on the repository could be `master`, `main`, `{current_version}.x` and any other,

It's safer to just let everyone declare their default branch, just like they would do with the original command.

Reply:
Yes, but I was assuming GitHub offers an easy way to retrieve that information from some king of context variable.
Maybe it's not possible, I am not very familiar with GitHub action.

Reply:
Done

Reply:
Done, @aglipanci could you review it??